src/utils/fetchTasks.ts

```diff
@@ -0,0 +1,16 @@
+import { FAILED_TO_FETCH_TASKS } from "../constants/responses";
+import { RDS_BASE_API_URL } from "../constants/urls";
+import { TasksResponseType } from "../typeDefinitions/task.types";
+
+async function fetchTasks(assignee: string, status: string) {
+  const url = `${RDS_BASE_API_URL}/tasks?status=${status}&assignee=${assignee}&dev=true`;
```

Comment:
is it possible to define return type  of this function here as well?

Reply:
not needed

Reply:
if possible you can make changes, otherwise no problem

Reply:
When assigning a value to a variable, there is no need to specify its data type.